### PR TITLE
mixer: add template CRD flag and set it to false

### PR DESCRIPTION
**Pull request by kyessenov**
Disable listening to custom instance CRs with a flag.

Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Comment by kyessenov** [comment]
/assign @douglas-reid 
/assign @geeknoid 

This is a follow-up to #12689 . I flipped the bit to not listen to template CRDs in helm template with a flag.

**Comment by kyessenov** [comment]
Confirmation from Mixer logs:
```
2019-03-27T21:15:05.571526Z	info	mcp	Requesting following collections:
2019-03-27T21:15:05.571559Z	info	mcp	  [0] istio/policy/v1beta1/instances
2019-03-27T21:15:05.571562Z	info	mcp	  [1] istio/policy/v1beta1/rules
2019-03-27T21:15:05.571565Z	info	mcp	  [2] istio/config/v1alpha2/templates
2019-03-27T21:15:05.571567Z	info	mcp	  [3] istio/config/v1alpha2/adapters
2019-03-27T21:15:05.571570Z	info	mcp	  [4] istio/policy/v1beta1/attributemanifests
2019-03-27T21:15:05.571573Z	info	mcp	  [5] istio/policy/v1beta1/handlers
```

File: mixer/cmd/mixs/cmd/server.go [hunk 1]
```diff
@@ -77,6 +77,8 @@ func serverCmd(info map[string]template.Info, adapters []adapter.InfoFn, printf,
 
 	serverCmd.PersistentFlags().BoolVar(&sa.UseAdapterCRDs, "useAdapterCRDs", sa.UseAdapterCRDs,
```
Comment (douglas-reid) [inline mixer/cmd/mixs/cmd/server.go]: i wonder: will we ever want to support a mix of these cases moving forward?

Comment (kyessenov) [reply mixer/cmd/mixs/cmd/server.go]: Probably not for actual users. We can delete them both together in the next release. The internal tests and test configs use a mix.

**Comment by kyessenov** [comment]
@mandarjog @geeknoid need an install approver...

**Pull request merged**